dandi/download.py

```diff
@@ -743,7 +744,7 @@ def _download_file(
     resuming = False
     attempt = 0
     attempts_allowed: int = (
-        3  # number to do, could be incremented if we downloaded a little
+        10  # number to do, could be incremented if we downloaded a little
     )
     while attempt <= attempts_allowed:
```

Comment:
This can be an off-by-one error, there will potentially attempts of 0, 1 ... `attempts_allowed` totalling `attempts_allowed + 1` attempts.

Reply:
done in the latest pushed commit -- will start with 1 and increment in finally